src/haddock/clis/restraints/calc_accessibility.py

```diff
@@ -214,10 +223,12 @@ def add_calc_accessibility_arguments(calc_accessibility_subcommand):
             'PNS': 78.11,
             }
     }
+DEFAULT_PROBE_RADIUS = 1.4
 
 
 def get_accessibility(
```

Comment:
best to move this inside the `sasascore` module and call it here instead of the other way around since these CLIs are marked for deletion

Reply:
I can move it to the restraints library, imo a better place than the module file